.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,55 @@
 <!--
 Thanks for sending a pull request!
 
-Please make sure you've read our contributing guidelines and how to submit a pull request (https://github.com/containers/podman/blob/master/CONTRIBUTING.md#submitting-pull-requests).
+Please make sure you've read our contributing guidelines and how to submit a pull request (https://github.com/containers/podman/blob/main/CONTRIBUTING.md#submitting-pull-requests).
 
 In case you're only changing docs, make sure to prefix the pull-request title with "[CI:DOCS]".  That will prevent functional tests from running and save time and energy.
+
+Finally, be sure to sign your commits (e.g. `git commit -s`)
+-->
+
+#### What this PR does / why we need it:
+
+<!---
+Please put your overall PR description here
+-->
+
+#### How to verify it
+
+<!---
+Please specify the precise conditions and/or the specific test(s) which must pass.
+-->
+
+#### Which issue(s) this PR fixes:
+
+<!--
+Please uncomment this block and include only one of the following on a
+line by itself:
+
+None
+
+-OR-
+
+Fixes #<issue number>
+
+*** Please also put 'Fixes #' in the PR title ***
+
+-->
+
+#### Special notes for your reviewer:
+
+#### Does this PR introduce a user-facing change?
```

Comment:
This is of somewhat limited utility because my existing release notes tooling works based on commit messages, not PRs (by necessity - our backports process makes identifying what commits are in a release, that were not in a previous release, quite complicated). I could theoretically work backwords from that to find the PR and get the release notes, but it would get time consuming (and I don't think it's easily scriptable)

Reply:
Okay, so we should just get rid of this section too then?

Reply:
Either that or tell them to add it directly to `RELEASE_NOTES.md`

Reply:
Oh! That's a novel idea, I like it!  But how could it be enforced?.  Humans would probably just ignore it in a template-comment :disappointed: 

I could see doing it by adding back in a `/kind` tagging mechanism.  Cirrus-CI supports running certain tests based on the PR labels.  For example, if a `kind:userfacing` tag is present, CI can make sure the`RELEASE_NOTES.md` file changed.  That way the requirement has "teeth".